api/tests/unit/environments/identities/test_unit_identities_models.py

```diff
@@ -662,19 +662,33 @@ def test_update_traits(environment: Environment) -> None:
     new_trait_1_value = 5
     trait_3_key = "trait_3"
     trait_3_value = 3
+
+    # and one transient trait that is evaluated against but not persisted
+    transient_trait_key = "transient"
+    transient_trait_value = "not persisted"
+
     trait_data_items = [
         generate_trait_data_item(
             trait_key=trait_1.trait_key, trait_value=new_trait_1_value
         ),
         generate_trait_data_item(trait_key=trait_3_key, trait_value=trait_3_value),
+        generate_trait_data_item(trait_key=trait_3_key, trait_value=trait_3_value),
```

Comment:
Hmm... it's odd to me that we're generating the same trait twice here?

Reply:
Fixed.